src/components/MapboxMap/index.tsx

```diff
@@ -48,11 +50,17 @@ const MapboxMap: React.FC<MapboxMapProps> = ({
   // Store map transform data in state so we can pass it to children
   const [transform, setTransform] = useState<MapboxMapTransform | null>(null);
 
-  // Let the parent component overrite the map bounds
-  usePropChangeEffect(fitBounds, () => {
+  // Let the parent component overwrite the map bounds
+  useEffect( () => {
```

Comment:
Ah so if this works I'm cool with it (and I guess there's no linter set up in this repo to complain anyway), but I do think we want to use [`useDeepCompareEffect`](https://github.com/kentcdodds/use-deep-compare-effect) instead— we only want to trigger this if the actual _values_ changes, not just the reference.

Reply:
See 9a282dfcef0bd65dc5a22590e9e0453885272862

Reply:
Oof, we get ```
Error: useDeepCompareEffect should not be used with dependencies that are all primitive values. Use React.useEffect instead.
```
for the center array. but we know that `[0,1] === [0,1]` evaluates to false.
see 7be4ac1